accounts/models.py

```
@@ -0,0 +1,8 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+class CustomUser(AbstractUser):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)